core/src/main/java/org/jdbi/v3/core/statement/SqlStatement.java

```diff
@@ -1217,6 +1217,66 @@ public final This bindList(String key, List<?> values) {
     }
 
     /**
+     * Bind a parameter for each value in the given list * number of property names,
+     * and defines an attribute as the comma-separated list of parameter references (using colon prefix).
+     *
+     * Used to create query similar to:
+     * select * from things where (id, foo) in ((1,'abc'),(2,'def'),(3,'ghi'))
+     * <p>
+     * Examples:
+     * <pre>
+     *
+     * List&lt;Things&gt; thingsToSearchBy = ...
+     * List&lt;Thing&gt; things = handle.createQuery("select * from things where (id, foo) in (&lt;thingKeys&gt;)")
```

Comment:
Yeah, I don't think there's a good fix here unless we improve the column name mapping further.  Ignore my comment here, it will just have to be duplicated for now :)